Assets/Scripts/PuddleOnEnter.cs

```diff
@@ -4,12 +4,15 @@
 
 public class PuddleOnEnter : MonoBehaviour
```

Comment:
this script doesn't look like it's being used? we could probably just delete it.